v1/website/static/js/code-blocks-buttons.js

```diff
@@ -44,4 +44,25 @@ window.addEventListener('load', function() {
       textEl.textContent = 'Copy';
     }, 2000);
   });
+
+  // add event listener for all tab
+  document.querySelectorAll('.nav-link').forEach(function(el) {
+    el.addEventListener('click', function(e) {
+      const groupId = e.target.getAttribute('data-group');
+      document
+        .querySelectorAll(`.nav-link[data-group=${groupId}]`)
+        .forEach(function(el) {
+          el.classList.remove('active');
+        });
+      document
+        .querySelectorAll(`.tab-pane[data-group=${groupId}]`)
+        .forEach(function(el) {
+          el.classList.remove('active');
```

Comment:
This shouldnt be placed here because not every docusaurus user has this file

Reply:
Hmm yeah I'm confused where to put it actually 😯

Reply:
Should I make something like this? https://gist.github.com/yangshun/55db997ed0f8f4e6527571fc3bee4675 and put all the related css outside of the lib?

Reply:
put it in v1/lib/static and modify the Head.js to always include it.

Reply:
@endiliey @JoelMarcey made the changes:D please review!